### DISCUSSION TITLE
Installing mediawiki2latex

### DISCUSSION THREAD

**Discussion by Fnuxus** [Homebrew/discussions#1246] [Getting started]
Hi! 

mediawiki2latex wouldn’t install on my Mac. Would anyone like to prepare it as a package for homebrew?

https://de.wikibooks.org/wiki/Benutzer:Dirk_Hünniger/wb2pdf/install

**Comment by SMillerDev** [comment]
Go ahead https://docs.brew.sh/Adding-Software-to-Homebrew

**Reply by carlocab** [reply]
Looks like it's got a LaTeX dependency, though, and I think that'll need the `mactex` cask. It might need to go to an external tap.

**Reply by Fnuxus** [reply]
> Go ahead https://docs.brew.sh/Adding-Software-to-Homebrew

Thanks for the link. I’ll look into it.
